### PR TITLE
feat: New datasource "sys_fs_cgroup_uniq_memory_swappiness" and its parser

### PR DESCRIPTION
**Pull request by qinpingli**
### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

*Add your description here*
This PR is used to check the memory.swappiness setting under `/sys/fs/cgroup/memory` dir, check the CEECBA-6282 to see the detailed information.

### CODE REVIEW

**Comment by qinpingli** [comment]
@xiangce please help review, thanks!

**Comment by qinpingli** [comment]
@xiangce I updated it to a datasource, please help review, thanks!

File: insights/specs/default.py
Comment (xiangce) [inline insights/specs/default.py:30]: Please adjust the line length accordingly.

File: insights/specs/datasources/sys_fs_cgroup_memory.py
Comment (xiangce) [inline insights/specs/datasources/sys_fs_cgroup_memory.py:23]: nitpick: I saw the number of spaces between the `value` and `count` here is different from others.

File: insights/parsers/sys_fs_cgroup_memory.py
Comment (xiangce) [inline insights/parsers/sys_fs_cgroup_memory.py:2]: Please update the title accordingly.

You may want to make the top title a bit rough, in case we can add other parsers related to "/sys/fs/cgroup/memory" into this file.

**Review by xiangce — CHANGES_REQUESTED** [review]

File: insights/parsers/sys_fs_cgroup_memory.py
Comment (xiangce) [inline insights/parsers/sys_fs_cgroup_memory.py:41]: should be `list`

**Review by xiangce — CHANGES_REQUESTED** [review]
Everything looks good, expect for the new introduced typo.

File: insights/parsers/sys_fs_cgroup_memory.py
Comment (qinpingli) [reply insights/parsers/sys_fs_cgroup_memory.py]: Sorry for the mistake, :(. 

Updated.

**Review by xiangce — APPROVED** [review]
Approved pending the approval of Spec

**Pull request merged**
